graphql/mappers/profile.js

```diff
@@ -8,51 +7,113 @@ export async function getProfileContent() {
   const mappedProfile = {
     en: {
       pageName: response.data.schPagev1ByPath.item.scTitleEn,
-      heading: response.data.schPagev1ByPath.item.scTitleEn,
-      cards: response.data.schPagev1ByPath.item.scFragments
-        .find((element) => element.scId === 'profile-cards')
-        .scItems.map((fragment) => {
-          return {
-            id: fragment.scId,
-            title: fragment.scTitleEn,
-            lists: {
-              tasks: fragment.schTasks.map((list) => {
+      heading: response.data.schPagev1ByPath.item.scFragments.find(
+        (element) => element.scId === 'profile-intro'
+      ).scContentEn.json[0].content[0].value,
+      list: response.data.schPagev1ByPath.item.scFragments
+        .map((element) => {
+          if (
+            element.scId !== 'profile-cards' &&
+            element.scId !== 'profile-intro' &&
+            element.scId !== 'looking-for-security-settings' &&
+            element.scId !== 'back-to-my-dashboard' &&
+            element.scId !== 'exit-beta-version'
+          ) {
+            return {
+              title: element.scTitleEn,
+              tasks: element.scItems.map((item) => {
                 return {
-                  title: list.scLinkTextEn,
-                  areaLabel: list.scLinkTextAssistiveEn,
-                  link: buildLink(list.schURLType, list.scDestinationURLEn),
-                  icon: list.scIconCSS,
-                  betaPopUp: list.schBetaPopUp,
+                  id: item.scId,
+                  title: item.scLinkTextEn,
+                  areaLabel: item.scLinkTextAssistiveEn,
+                  link: buildLink(item.schURLType, item.scDestinationURLEn),
+                  icon: item.scIconCSS,
+                  betaPopUp: item.schBetaPopUp,
                 }
               }),
-            },
+            }
           }
         })
         .filter((e) => e),
+      lookingFor: {
+        title: response.data.schPagev1ByPath.item.scFragments.find(
+          (element) => element.scId === 'looking-for-security-settings'
+        ).scContentEn.json[0].content[0].value,
+        subText: [
+          response.data.schPagev1ByPath.item.scFragments.find(
+            (element) => element.scId === 'looking-for-security-settings'
```

Comment:
Can we define the data for certain components in a constant at the top of the page like on the security-settings page. This will keep us from running multiple find fragment calls and will probably make it easier to error proof in the future. 
ex. (we may be able to do this without separating en and fr into two calls)
```
const enLookingForFragment = findFragmentByScId(
    response,
    'looking-for-profile-settings'
  ).scContentEn
```

Reply:
Should be good!